### ISSUE TITLE
android.testOptions.junitPlatform.unitTests.includeAndroidResources does not work

### ISSUE THREAD

**Issue by theojalba** [mannodermaus/android-junit5#78]
When I add the option `android.testOptions.junitPlatform.unitTests.includeAndroidResources` to the app `build.gradle`, there is an Gradle error:

```
Could not set unknown property 'includeAndroidResources' for object of type de.mannodermaus.gradle.plugins.junit5.AndroidJUnitPlatformExtension$UnitTestOptions.
```

This is a snippet from the `build.gradle`:

```
android {
  ...
  testOptions {
    ...
   junitPlatform {
     ...
     unitTests {
       // Fix unit test resources not included with AS3
       includeAndroidResources = true
     }
  }
}
```

Some background info on this flag: https://android.jlelse.eu/make-robolectric-compatible-with-latest-gradle-tools-3-0-0-eca085681b3b

To fix this in the plugin, it seems you need to add the following line:
`var includeAndroidResources = false`

around here:
https://github.com/mannodermaus/android-junit5/blob/715344a0afba49370979e20f614a1904ba688848/android-junit5/src/main/kotlin/de/mannodermaus/gradle/plugins/junit5/Dsl.kt#L523

Could you fix this and release a new version with support for this flag so we can use resources in unit tests?

**Comment by theojalba** [comment]
Opened PR here: https://github.com/mannodermaus/android-junit5/pull/79

**Comment by mannodermaus** [comment]
Thanks for reporting! As you already noticed, there is no support for Android resources in unit tests right now, so the DSL doesn't expose that field. I'll target this for the next release, but it's going to be more than just adding that flag. Not sure when I'll get around to doing the implementation. I'll leave some more comments on your PR if you want to give it a go.

**Comment by mannodermaus** [comment]
This was implemented by #80. Thanks again for reporting! Please try it out with the latest `1.0.32-SNAPSHOT`!
